Python/instrumentation.c

```diff
@@ -711,7 +717,22 @@ remove_tools(PyCodeObject * code, int offset, int event, int tools)
     assert(event != PY_MONITORING_EVENT_LINE);
     assert(event != PY_MONITORING_EVENT_INSTRUCTION);
     assert(PY_MONITORING_IS_INSTRUMENTED_EVENT(event));
-    assert(opcode_has_event(_Py_GetBaseOpcode(code, offset)));
```

Comment:
This is also correct, provided the assertion `assert(PY_MONITORING_IS_INSTRUMENTED_EVENT(event))` is true.

Reply:
Hmm, it will guarantee that the opcode is not ENTER_EXECUTOR?